fetch.bs

```diff
@@ -3335,6 +3340,9 @@ optionally with a <i>recursive flag</i>, run these steps:
    <li><p><a href=https://w3c.github.io/webappsec-upgrade-insecure-requests/#upgrade-request>Upgrade <var>request</var> to a potentially secure URL, if appropriate</a>.
    [[!UPGRADE]]
 
+   <li><p><a href=https://w3c.github.io/webappsec-fetch-metadata/#abstract-opdef-set-the-fetch-metadata-headers-for-a-request>Set the Fetch metadata headers for <var>request</var></a>.
```

Comment:
This should be an exported term. You can file a bug against tabatkins/bikeshed to get Fetch Metadata indexed. If all goes well we can land this PR Monday.

Reply:
Hrm. It is exported from Fetch Metadata, and that spec shows up in Bikeshed's biblio entries, but not in its anchor index. Weird!

Filed https://github.com/tabatkins/bikeshed/issues/1584

Reply:
Biblio is for references at the end of the document, Shepherd is for inline cross-references and does not pick up new specifications to crawl automatically.